traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+OSI only specifies the interface of the traffic participant.
+OSI is agnostic to the architecture of the simulation environment and the traffic participant.
+
+
+Use cases for traffic participants
+----------------------------------
+
+Different models are involved in modeling a traffic participant.
+In all the use cases, a simulator loads and interprets a scenario and a map prior to execution.
+The scenario is, for example, provided by OpenSCENARIO.
+The map data is, for example, provided by OpenDRIVE.
+During runtime the simulator interacts with the traffic participants via OSI messages.
+There can be multiple instances of a traffic participant.
+The traffic participants are co-simulated.
+
+The following figure shows a very simple use case.
+
+.. image:: _static/images/osi-traffic-participant-use-case-1.png
+   :align: center
+   :alt: Simple traffic participant
+
+The traffic participant bases its behavior only on an idealized view of the area around it.
+If they exist, the traffic participant’s dynamics are included in the model.
+The map, the simulated world state, the sensor view, and the traffic updates represent the ground truth, meaning objective statements about the world.
+The scenario and the traffic commands on the other hand represent the intentions of the traffic participants.
+
+The following figure shows a traffic participant with separately modeled behavior and dynamics.
+
+.. image:: _static/images/osi-traffic-participant-use-case-2.png
+   :align: center
+   :alt: Traffic participants with separate dynamics
+
+Separating dynamics from behavior introduces two more types of channels:
+
+* Measured data, meaning ground truth as interpreted by the traffic participant
+* Actuator-intention data, providing information on what actuators are supposed to do
+
+OSI currently provides only limited support for data structures describing measured internal states of the traffic participant.
+Actuator intentions are currently not covered by OSI and must be handled with a different data description format.
+
+The following figure shows a more complex traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-use-case-3.png
+   :align: center
+   :alt: Traffic participant with sensor models, AD function, and dynamic model
+
+This use case will probably be relevant for modeling the ego vehicle, which includes the system-under-test.
+The traffic participant includes an arbitrary number of sensor models, which provide sensor data.
+Sensor data is another example of measured data as an interpretation of ground truth by the traffic participant.
+In contrast to data describing the internal state of the vehicle, sensor data describes how sensors interpret the surroundings, for example, with lidar, camera, radar, or ultrasonic technology.
+The AD function consumes measured data and outputs actuator-intention data.
+OSI currently does not support actuator-intention data.
```

Comment:
In consistency with the comment above I think this paragraph also needs to be rephrased/shortened. Suggestion: "The traffic participant includes an arbitrary number of sensor models, consuming sensor view and providing sensor data. The AD function consumes this sensor data. The further data flow to a dynamics model is currently not supported by OSI. Yet the loop to the simulator can be closed via Traffic update."

Reply:
- [ ] Rework paragraph.